File: benchmark/results/java-invocation-inline/run-005/b-baseline/output/com/metaengine/demo/ordering/services/OrderValidator.java
package com.metaengine.demo.ordering.services;

import com.metaengine.demo.ordering.aggregates.Order;
import java.util.List;

/** Validation service for the Order aggregate. */
public class OrderValidator {

    /** Validate and create a new Order from a partial input. */
    public Order create(Order input) {
        throw new UnsupportedOperationException("not implemented");
    }

    /** Find an Order by id; returns null if not found. */
    public Order findById(String id) {
        throw new UnsupportedOperationException("not implemented");
    }

    /** List Orders, capped at the supplied limit. */
    public List<Order> list(int limit) {
        throw new UnsupportedOperationException("not implemented");
    }

    /** Delete an Order by id. */
    public void delete(String id) {
        throw new UnsupportedOperationException("not implemented");
    }
}

File: benchmark/results/ts-multilang/run-002/a-mcp/output/src/domain/ordering/services/order-validator.ts
import { Order } from '../aggregates/order';

/** OrderValidator service. */
export class OrderValidator {

  create(input: Partial<Order>): Order { throw new Error('not implemented'); }

  findById(id: string): Order | null { throw new Error('not implemented'); }

  list(limit: number): Order[] { throw new Error('not implemented'); }

  delete(id: string): void { throw new Error('not implemented'); }
}
